src/main/java/com/juniorstart/juniorstart/util/ModelMapper.java

```diff
@@ -0,0 +1,118 @@
+package com.juniorstart.juniorstart.util;
+
+import com.juniorstart.juniorstart.model.*;
+import com.juniorstart.juniorstart.payload.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+
+/** Util class to map a model class to a model responses class.
+ * @author Grzegorz Szczęsny
+ * @version 1.0
+ * @since 1.0
+ */
+public class ModelMapper {
+
+
+
+    /** Map Project's object to a ProjectResponse class, which is send to response.
+     * @param project The object that represents a project's credentials.
+     * @param user The object that represents a user's credentials.
+     * @param userProjectLike The like's amount on project.
+     * @return a {@link ProjectResponse} that is send to a frontend.
+     */
+    public static ProjectResponse mapProjectToProjectResponse(Project project, User user, long userProjectLike) {
+
+        ProjectResponse projectResponse = new ProjectResponse();
+        projectResponse.setId(project.getId());
+        projectResponse.setTitle(project.getTitle());
+        projectResponse.setDescription(project.getDescription());
+        projectResponse.setBody(project.getBody());
+        projectResponse.setNumberOfSeats(project.getNumberOfSeats());
+        projectResponse.setRepository(project.getRepository());
+        projectResponse.setName(project.getName());
+        projectResponse.setLikes(userProjectLike);
+
+             List<ProjectTodoResponse> projectTodoResponses = project.getProjectTodos().stream().map(projectTodo -> {
+            return ModelMapper.mapProjectTodoToProjectTodoResponse(projectTodo, projectTodo.getUser());
+        }).collect(Collectors.toList());
+
+             List<TechnologyResponse> technologyResponses = project.getTechnologies().stream().map(technology -> {
+                 return ModelMapper.mapTechnologyToTechnologyResponse(technology);
+             }).collect(Collectors.toList());
+
+
+        List<UserSummary> team_members = project.getTeam_members().stream().map(member -> {
+            return UserSummary.builder().id(member.getPublicId()).name(member.getName())
+                    .email(member.getEmail()).build();
+        }).collect(Collectors.toList());
+
+        List<ProjectCommentResponse> comments = project.getProjectComments().stream().map(projectComment -> {
+            return ModelMapper.mapProjectCommentToProjectCommentResponse(projectComment, user);
+        }).collect(Collectors.toList());
+
```

Comment:
Jak używasz buildera to każda kropka pod nowej linijki. Robi się to bardziej czytelne.